cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -117,35 +113,45 @@ mkTracers :: forall peer blk.
            -> Tracers peer blk
 mkTracers traceOptions tracer = Tracers
     { chainDBTracer
-        = annotateSeverity $ filterSeverity (pure . const (tracingSeverity $ traceChainDB traceOptions))
+        = tracerOnOff (traceChainDB traceOptions)
+          $ annotateSeverity $ filterSeverity (pure . const (tracingSeverity True))
```

Comment:
the calls to `filterSeverity` on something that is "True" can be removed. previously, the flag was used to control the filter.